packages/suite/src/hooks/wallet/sign-verify/useSignVerifyForm.ts

```diff
@@ -64,9 +43,21 @@ const signVerifySchema = yup.object<ValidationSchema<Omit<SignVerifyFields, 'isE
         is: false,
         then: schema => schema.required(),
     }),
-    hex: yup.boolean(),
+    hex: yup.boolean().required(),
```

Comment:
This relates to the message and therefore shall be required for the very same reason as message is.